exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet2/DrillParquetReader.java

```diff
@@ -84,14 +82,7 @@
   private RecordReader<Void> recordReader;
   private DrillParquetRecordMaterializer recordMaterializer;
   private int recordCount;
-  private List<ValueVector> primitiveVectors;
   private OperatorContext operatorContext;
```

Comment:
Can you explain the reason you are removing the existing logic?

Reply:
Variable was not used.